### DISCUSSION TITLE
Question about Modal and Browser Conflict Issue

### DISCUSSION THREAD

**Discussion by kam5263** [twbs/bootstrap#41048] [Q&A] [answered]
When using a modal in Bootstrap, I encounter the following error message in the browser console after clicking the close button:

```
Blocked aria-hidden on an element because its descendant retained focus. The focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus. For more details, see the aria-hidden section of the WAI-ARIA specification at https://w3c.github.io/aria/#aria-hidden.
Element with focus: button
Ancestor with aria-hidden:
```
Even when I run the sample code provided in the Bootstrap 5.3 documentation (https://getbootstrap.com/docs/5.3/components/modal/#modal-components), the same issue occurs. Could you help me understand the reason behind this error?

Thank you in advance!

**Comment by julien-deramond** [comment, marked as answer]
It must be linked to https://github.com/twbs/bootstrap/issues/41005
